sdk/go/common/resource/plugin/provider_plugin.go

```diff
@@ -181,6 +181,24 @@ func NewProvider(host Host, ctx *Context, pkg tokens.Package, version *semver.Ve
 		if jsonConfig != "" {
 			env = append(env, "PULUMI_CONFIG="+jsonConfig)
 		}
+
+		// TODO: where should this config come from?
```

Comment:
I am not sure what the best approach is to thread this information through to here. Via `Host`?